mailbox/backup/src/test/java/org/apache/james/mailbox/backup/ZipAssertTest.java

```diff
@@ -59,30 +63,21 @@ void beforeEach(TemporaryFolderExtension.TemporaryFolder temporaryFolder) throws
         ExtraFieldUtils.register(SizeExtraField.class);
     }
 
-    @Test
-    public void hasNoEntryShouldNotThrowWhenEmpty() throws Exception {
+    private void buildZipFile(List<SimpleImmutableEntry<String, byte[]>> entries) throws Exception {
```

Comment:
you could use a vararg to make the caller job easier

Reply:
Thanks, I didn't heard about that. I will check the effective java book on this subject. :)